great_expectations/expectations/expectation.py

```diff
@@ -1552,14 +1559,14 @@ def _get_renderer_diagnostics(
         expectation_type: str,
         test_diagnostics: List[ExpectationTestDiagnostics],
         registered_renderers: List[str],
-        standard_renderers: List[str] = [
-            "renderer.answer",
-            "renderer.diagnostic.unexpected_statement",
-            "renderer.diagnostic.observed_value",
-            "renderer.diagnostic.status_icon",
-            "renderer.diagnostic.unexpected_table",
-            "renderer.prescriptive",
-            "renderer.question",
+        standard_renderers: List[Enum] = [
```

Comment:
```suggestion
        standard_renderers: List[Union[LegacyRendererType, LegacyDiagnosticRendererType]] = [
```

Can we use a `Union` of the renderers?
If this is something that might be re-used it's probably better to define it in the someplace we define the enums themselves.


```python

class LegacyRendererType(Enum):
   ...

class LegacyDiagnosticRendererType(Enum):
   ...

AnyRendererType = Union[LegacyRendererType, LegacyDiagnosticRendererType]

```
Or if only specific renderers are supported we could use a `Literal`, although I suspect this won't be needed.
https://mypy.readthedocs.io/en/stable/literal_types.html#literal-types-and-enums

Reply:
✅